src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ReportStatusCode.java

```diff
@@ -1,10 +1,12 @@
 package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
 
 import java.util.Arrays;
+import java.util.Map;
 
 import lombok.Getter;
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
+import org.hl7.fhir.dstu3.model.ProcedureRequest;
```

Comment:
This is an edifact model, so there's no reason for it to know anything about fhir imo

Reply:
Is it better now? 818a10323f2cbd4f2ab584b1908df2510e7a4d14 Thanks!